### DISCUSSION TITLE
Binary distribution on a Github repository

### DISCUSSION THREAD

**Discussion by tallero** [fraunhoferhhi/vvenc#494] [Q&A]
Hello,
I've just read the BSD-3-clause-clear license under which this software is released, and there's something I haven't got.
I am allowed to have people download it from a software repository of mine if I do specify somewhere the authors of this software do not endorse in any way my software repository?
Also I find relatively weird that apparently I'm able to promote my software repository using this software but not using the name of the copyright holder?
Why should I promote the software repository using the name of the copyright holder instead of the software itself?
I mean, it's not like people usually care about the authors of things, but rather about the things.
Or it's more like it's considered endersement or promotion already that the author's and contributors' name is included somewhere in the repository so basically the binary distribution is not allowed at all without prior permission?

**Comment by adamjw24** [comment]
> I am allowed to have people download it from a software repository of mine if I do specify somewhere the authors of this software do not endorse in any way my software repository?

No need to put that. Just don't say we endorse it or that is "an official binary distribution" or similar and should be ok.

> Also I find relatively weird that apparently I'm able to promote my software repository using this software but not using the name of the copyright holder?

VVenC as a name stands for itself, leave the authors out of it. You are distributing the copy of the software, so use its name to identify it. Why would you include the authors or copyright holders anywhere if they are not involved there?

> Why should I promote the software repository using the name of the copyright holder instead of the software itself? I mean, it's not like people usually care about the authors of things, but rather about the things.

Exactly, just stick to identyfying your binary as VVenC and you should be ok. "Fraunhofer Versatile Video Encoder" is also somehow a name we use so that should also be ok - its on us to have included the name of copyright holders in the name, you would only be using the name.

> Or it's more like it's considered endersement or promotion already that the author's and contributors' name is included somewhere in the repository so basically the binary distribution is not allowed at all without prior permission?

What you say is attribution, not endorsement. Just don't put "the official binary repository of Fraunhofer HHI for VVenC" and you should be fine. Somewhere around your download tho, or in the binary distribution itself, the copy of the license and the authors list should be available.

Anyhow, I am not a lawyer and my comment is no way binding. We decided to use standard license in the hope there are standard interpretations of this license, so please put some work to read through those.
